firedrake/utility_meshes.py

```diff
@@ -492,6 +492,7 @@ def OneElementThickMesh(
 @PETSc.Log.EventDecorator()
 def UnitTriangleMesh(
     comm=COMM_WORLD,
+    refinement_level=0,
```

Comment:
If we can refine, I think we should also add `distribution_parameters` as argument just as in other functions.

Reply:
Added in last commit

Reply:
Could you also add docstring for this:
```
:kwarg distribution_parameters: options controlling mesh
distribution, see :func:`.Mesh` for details.
```

Reply:
Added